R/conf_mat.R

```diff
@@ -313,11 +313,14 @@ cm_heat <- function(x) {
   `%+%` <- ggplot2::`%+%`
 
   as.data.frame.table(x$table) %>%
-    ggplot2::ggplot(ggplot2::aes(Prediction, Truth, fill = Freq)) %+%
+    ggplot2::ggplot(ggplot2::aes(x = Prediction,
```

Comment:
Would it be slightly better to do:

```
dplyr::mutate(Truth = factor(Truth, levels = rev(levels(Truth))) %>%
```

right before the ggplot2 call so then you don't need to update the label at the end?